source/Canvas.js

```diff
@@ -17,6 +17,7 @@ class Canvas {
 
   computeCartogram(options) {
     this._mapGraphic.computeCartogram(options)
+    this.getCartogramArea()
```

Comment:
Yeah good catch. I think I meant to set it on the canvas object here and then call it when the slider changed. Fixing.